ami/users/api/views.py

```
@@ -0,0 +1,86 @@
+from ami.users.api.serializers import (
+    ProjectRoleSerializer,
+    UserProjectMembershipListSerializer,
+    UserProjectMembershipSerializer,
+)
+from ami.users.roles import Role
+
+logger = logging.getLogger(__name__)
+
+
+class RolesAPIView(APIView):
+    def get(self, request, **kwargs):
```

Ruff (0.14.8)

Unused method argument: `request`

(ARG002)